src/basics/building-deploying-interfacing.md

```diff
@@ -1,17 +1,17 @@
 # Building, deploying and interfacing
 
-Now that all of the dependencies are installed and we have successfully chosen a [development track], we can finally discuss program development, including compiling, deploying and interacting with it.
+Now that all of the dependencies are installed and we have successfully chosen a [development stack], we can finally discuss program development, including compiling, deploying and interacting with it.
```

Comment:
I think @waymobetta meant track because there were two different tracks in the documentation, but I'm not sure if that's still accurate.

Reply:
I'm good with this if you're modifying this to refer to them as stacks as I'm not too picky on the nomenclature.

Reply:
https://github.com/Perelyn-Arch/arch-book/commit/f29577bf981fc0231e518b7992477ffad0975ff5